linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_command/rule.yml

```diff
@@ -23,6 +23,11 @@ severity: medium
 identifiers:
     cce@rhel8: 80940-0
 
+references:
+    disa: 56,58
+    nist: AC-11(a),AC-11(b)
```

Comment:
(just documenting the reasoning)

- AC-11(a) because of the "upon receiving a request from a user" part;
- AC-11(b) because default behavior is to force re-authentication

Reply:
right... thanks.